src/SFA.DAS.Activities.sln

```diff
@@ -1,10 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26730.16
+VisualStudioVersion = 15.0.27130.2027
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{CC5FD16D-436D-48AD-A40C-5A424C6E3E79}") = "SFA.DAS.Activities.CloudService", "SFA.DAS.Activities.CloudService\SFA.DAS.Activities.CloudService.ccproj", "{291916AB-7AC4-45B9-9112-D9D846401E66}"
-EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SFA.DAS.Activities.Worker", "SFA.DAS.Activities.Worker\SFA.DAS.Activities.Worker.csproj", "{ED9598CD-DDE8-453C-B36B-68033A453FE6}"
```

Comment:
Going with our new project naming conventions can this be renamed to `SFA.DAS.Activities.MessageHandlers`?

Reply:
Done